content/doc/book/managing/cli.adoc

```diff
@@ -242,6 +242,26 @@ You can also precede the argument with `@` to load the same content from a file:
 java -jar jenkins-cli.jar [-s JENKINS_URL] -auth @/home/kohsuke/.jenkins-cli command ...
 ----
 
+The third and last authentication method is to configure environment variables in similar way as the `$JENKINS_URL`
```

Comment:
I would not say it's the last authentication method. At least not in general, but only for HTTP. The next paragraph is about SSH authentication, that sounds weird.

Reply:
[windows user testimonials] It's weird to see environment variables beginning with `$` (that's the case for the rest of the document anyway, i.e. nothing to change)

Reply:
> I would not say it's the last authentication method. At least not in general, but only for HTTP. The next paragraph is about SSH authentication, that sounds weird.

Makes sense. I'll change it

> [windows user testimonials] It's weird to see environment variables beginning with $ (that's the case for the rest of the document anyway, i.e. nothing to change)

I'm linux user, but I absolutely agree.